MAUI/Polar-Charts/Axis/Overview.md

```diff
@@ -0,0 +1,25 @@
+---
+layout: post
+title: About Axis in .NET MAUI Chart control | Syncfusion
+description: Learn here all about the chart axis and its key features in the Syncfusion .NET MAUI Chart (SfPolarChart) control.
+platform: maui
+control: SfPolarChart
+documentation: ug
+---
+
+# Axis in .NET MAUI Chart
+
+The [ChartAxis]() is used to locate a data point within the chart area. Charts usually consist of two axes, which are used to measure and categorize data: a primary axis and a secondary axis.
+
+Polar chart supports the following types of chart axis.
+
+* Category
+* Numeric
+* Date time
+
+## Key features
+
+* Axis supports different types of axes such as Category, Numeric, and Date Time.
+* It allows you to customize the chart axis line, including grid lines and tick lines. You can also customize the style of the axis line by defining the [AxisLineStyle]().
```

Comment:
* It allows you to customize the chart axis line, including grid lines, and tick lines. You can also customize the style of the axis line by defining the [AxisLineStyle]().